teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
```

Comment:
- [ ] parent_permission should be included in the arguments for the initialize method as well.